tests/components/alexa/test_smart_home.py

```diff
@@ -58,7 +54,8 @@ def test_create_api_message_defaults(hass):
 
     assert msg["header"]["messageId"] is not None
     assert msg["header"]["messageId"] != directive_header["messageId"]
-    assert msg["header"]["correlationToken"] == directive_header["correlationToken"]
+    assert msg["header"]["correlationToken"] == directive_header[
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)